apps/meteor/client/views/room/UserCard/UserCardWithData.tsx

```diff
@@ -25,6 +26,9 @@ const UserCardWithData = ({ username, rid, onOpenUserInfo, onClose }: UserCardWi
 	const showRealNames = Boolean(useSetting('UI_Use_Real_Name'));
 
 	const { data, isLoading } = useUserInfoQuery({ username });
+	const { data: isMemberData, refetch, isSuccess: membershipCheckSuccess } = useMemberExists({ roomId: rid, username });
+
+	const isMember = (isMemberData?.exists as boolean) && membershipCheckSuccess;
```

Comment:
We should not just ignore the errors or the error-handling mechanism provided by useQuery. We must handle cases like when this is still loading, or if an error happens. Otherwise, this can create even flaky test cases because while the component is still loading we are showing some options, when the request is done we show other ones instead, making the screen flicker.